maintenance/maintenance.html

```diff
@@ -28,11 +28,6 @@
         <div class="takeover"></div>
         <div class="wrapper">
             <div class="sticky-top">
-                <div class="wrapper-join-us">
```

Comment:
We never load our compiled JS file into this page which means the old React component for rendering newsletter sign form never worked.